### PR TITLE
Update GoBuster User Guide

### PR DESCRIPTION
**Pull request by akshatsoni64**

### CODE REVIEW

**Review by Oliver2049 — COMMENTED** [review]
Hi @akshatsoni64 
Overall, it's good and works well when testing. However, I noticed that you forgot to remove outdated component (UserGuide) and you also forgot to add  import { LoadingOverlayAndCancelButtonPkexec } from "../OverlayAndCancelButton/OverlayAndCancelButton"; 
Regards,
Oliver

**Comment by michaeljpigott** [comment]
Hi @Oliver2049 can you please submit a review for @akshatsoni64 and request changes? 

Kind regards

Michael

**Comment by akshatsoni64** [comment]
> Hi @akshatsoni64 Overall, it's good and works well when testing. However, I noticed that you forgot to remove outdated component (UserGuide) and you also forgot to add import { LoadingOverlayAndCancelButtonPkexec } from "../OverlayAndCancelButton/OverlayAndCancelButton"; Regards, Oliver

Hi Oliver,
I have just updated the message displayed on the User Guide Page, I don't know much about React, and hence, trying to avoid breaking the code. Do you think those updates are absolutely necessary to resolve the issue?

File: src/components/GobusterTool/Gobuster.tsx
Comment (Oliver2049) [inline src/components/GobusterTool/Gobuster.tsx:1]: You should remove this one
```suggestion

```

**Review by Oliver2049 — APPROVED** [review]
Hi @akshatsoni64 
Overall, it looks good to me. Thanks for your effort.
regards
Oliver

**Pull request merged**